easybuild/easyblocks/w/wien2k.py

```diff
@@ -186,23 +204,57 @@ def configure_step(self):
+                 'Please specify the ROOT-path of your FFTW installation (like /opt/fftw3):': fftw_root,
+                 'is this correct? enter Y (default) or n:': 'Y',
+                 'LIBXC (that you have installed before)? (y,N):': '',
             })
+
+            if LooseVersion(self.version) >= LooseVersion("17"):
+                qanda.update({
+                        'LIBXC (that you have installed)? (y,N):': '',
+                        'You need to KNOW details about your installed MPI, ELPA, and FFTW ) (y/N)': 'y',
+                        'Do you want to use a present ScaLAPACK installation? (Y,n):': 'y',
+                        'Do you want to use the MKL version of ScaLAPACK? (Y,n):': 'n',  # we set it ourselves below
+                        'Do you use Intel MPI? (Y,n):': 'y',
+                        'Is this correct? (Y,n):': 'y',
+                        'Please specify the target architecture of your ScaLAPACK libraries (e.g. intel64)!:': '',
+                        'ScaLAPACK root:': os.getenv('MKLROOT') or os.getenv('EBROOTSCALAPACK'),
+                        'ScaLAPACK library:': next((lib[2:] for lib in os.getenv('LIBSCALAPACK').split()
+                                                    if 'scalapack' in lib), 'scalapack'),
+                        'BLACS root:': os.getenv('MKLROOT') or os.getenv('EBROOTOPENBLAS'),
+                        'BLACS library:': next((lib[2:] for lib in os.getenv('LIBSCALAPACK').split()
+                                                if 'blacs' in lib), 'openblas'),
+                        'Please enter your choice of additional libraries!:': '',
+                        'Do you want to use a present FFTW installation? (Y,n):': 'y',
+                        'Please specify the path of your FFTW installation (like /opt/fftw3/) '
+                        'or accept present choice (enter):': fftw_root,
+                        'Please specify the target achitecture of your FFTW library (e.g. lib64) '
+                        'or accept present choice (enter):': '',
+                        'Do you want to automatically search for FFTW installations? (Y,n):': 'n',
+                        'Please specify the ROOT-path of your FFTW installation (like /opt/fftw3/) '
+                        'or accept present choice (enter):': fftw_root,
+                        'Is this correct? enter Y (default) or n:': 'Y',
+                        'Do you want to use ELPA? (y,N):': 'N',
+                        'Please specify the name of your FFTW library or accept present choice (enter):': '',
+                        'Please specify your parallel compiler options or accept the recommendations '
+                        '(Enter - default)!:': '',
+                        'Please specify your MPIRUN command or accept the recommendations (Enter - default)!:': '',
+                        'Please enter the full path to your temporary directory:': '/tmp',
```

Comment:
@migueldiascosta please don't hardcode `/tmp` here, use `tempfile.gettempdir()` (which was set to a unique directory for this `eb` session)

Reply:
this will be hardcoded into the `lapw` scripts, so it cannot be an `eb` temporary directory

I think the `siteconfig_lapw` requires a real path here... but maybe be can patch the scripts afterwards to use a variable...

Reply:
OK, please mention that in a comment, that this is a directory that must exist at runtime...